.pipelines/templates/template-run-rp-and-e2e.yml

```diff
@@ -0,0 +1,81 @@
+parameters:
+  workingDirectory: ""
+steps:
+  - script: |
+      cd ${{ parameters.workingDirectory }}
+
```

Comment:
is it not worth merging this with hack/e2e.sh so that we can also run the test easily locally?